src/dev-11/main/java/by/epam/training/bean/Draughts.java

```diff
@@ -0,0 +1,57 @@
+package by.epam.training.bean;
+
+/**
+ * Class draughts witch includes private fields
+ */
+public class Draughts {
+    private String color;
```

Comment:
It is better to store start and finish positions as int coordinates, parse string to get it.